docs/development/dev-container-setup.md

```diff
@@ -0,0 +1,170 @@
+# Development Container Setup Guide
+
+This guide explains how to set up and use the development environment using a Dev Container. The Dev Container provides a consistent and reproducible environment for development, testing, and building the Scality COSI Driver.
+
+---
+
+## Prerequisites
+
+- **Visual Studio Code** with the **Remote - Containers** extension.
```

Comment:
Maybe worth adding a link to the extension.

Reply:
I will add this.